src/CMakeLists.txt

```diff
@@ -191,7 +191,7 @@ elseif(${CMAKE_CXX_COMPILER_ID} STREQUAL "GNU")
     # GCC's arry bounds, maybe uninitialized, and restrict warning seems to have false positives.
     target_compile_options(lexy_dev INTERFACE -Wno-array-bounds -Wno-maybe-uninitialized -Wno-restrict)
 elseif(MSVC)
-    target_compile_options(lexy_dev INTERFACE /WX /W3 /D _CRT_SECURE_NO_WARNINGS /wd5105)
+    target_compile_options(lexy_dev INTERFACE /WX /W3 /D _CRT_SECURE_NO_WARNINGS /wd5105 /EHsc /utf-8)
```

Comment:
What is the error /EHsc fixes?

Reply:
The corresponding errror message is:

```
[5/301] Building CXX object examples\CMakeFiles\lexy_example_protobuf.dir\protobuf.cpp.obj
FAILED: examples/CMakeFiles/lexy_example_protobuf.dir/protobuf.cpp.obj 
C:\PROGRA~1\MIB055~1\2022\COMMUN~1\VC\Tools\MSVC\1437~1.328\bin\Hostx64\x64\cl.exe  /nologo /TP 
    -IC:\Users\shohi\Documents\github\lexy\src\..\include /utf-8 /MDd /Zi /Ob0 /Od /RTC1 -std:c++20 /WX /W3 /D 
    _CRT_SECURE_NO_WARNINGS /wd5105 /utf-8 
    /showIncludes /Foexamples\CMakeFiles\lexy_example_protobuf.dir\protobuf.cpp.obj 
    /Fdexamples\CMakeFiles\lexy_example_protobuf.dir\ /FS -c C:\Users\shohi\Documents\github\lexy\examples\protobuf.cpp
C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\vector(835): error C2220: warning treated as error - no object file generated
C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822\include\vector(835): warning C4530: C++ exception handler used, but unwind semantics are not enabled. Specify /EHsc
```

Reply:
Hm, I have a hard time believing that this is default CMake behavior. All sources I could find point to "/EHsc" being applied by default. Could this apply to you perhaps? https://github.com/microsoft/verona/issues/49#issuecomment-576457428

Reply:
It seems relevant to my problem.
Altough the error only occured when opening the project using Visual Studio, it suddenly dissapeared after I updated VS from 17.7.5 to 17.7.6 a few days ago... strange.
I think we do not have to put /EHsc option now as you mentioned.

Reply:
Okay. Can you do a force push with two commits: One that fixes the typo and one thats add /utf-8?